### ISSUE TITLE
Merging rem rules sometimes removes pixel fallback, even with compatibility/skip aggressive flags

### ISSUE THREAD

**Issue by kinglozzer** [jakubpawlowicz/clean-css#558]
Take the following example:

```css
.class {
  font-size: 11px;
  font-size: 1.1rem;
  margin: 0 0 1em; /* Note, merging works if this rule is removed */
}

.class strong {
  font-weight: bold;
}

.class {
  font-size: 14px;
  font-size: 1.4rem;
}
```

`cleancss -o test.min.css test.css --compatibility ie8 --skip-agressive-merging`

Output is:
```css
.class{margin:0 0 1em;font-size:1.4rem}.class strong{font-weight:700}
```

Tested on v3.2.6.

**Comment by kinglozzer** [comment]
Setting the `--skip-advanced` option works, so it’s something in the advanced merging. Possible duplicate of #363, although that was (incorrectly?) closed in 13e6752e20dbeee45528aa11353a1020ba9a89f3.

**Comment by jakubpawlowicz** [comment]
@kinglozzer thanks for reporting, I'll take a closer look in the evening - may be a regression from 3.1.x.

**Comment by jakubpawlowicz** [comment]
@kinglozzer It's fixed in 3.2.7.

**Comment by kinglozzer** [comment]
:+1: thanks @jakubpawlowicz
